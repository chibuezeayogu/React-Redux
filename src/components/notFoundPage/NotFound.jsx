import React from 'react';

const NotFound = () => (
	<div className="card text-center not-found">
		<div class="card-header"/> 
		<div className="card-body">
			<h2 className="card-title">Page Not Found</h2>
		</div>
		<div className="card-footer text-muted"/>
	</div>
);

export default NotFound;
