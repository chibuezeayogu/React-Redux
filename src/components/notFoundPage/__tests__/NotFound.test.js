import React from 'react';
import { shallow } from 'enzyme';

import NotFound from '../NotFound';


describe('Header Component', () => {
	const wrapper = shallow(<NotFound />);

	it('should render correctly', () => {
		expect(wrapper).toMatchSnapshot();
	});
});
