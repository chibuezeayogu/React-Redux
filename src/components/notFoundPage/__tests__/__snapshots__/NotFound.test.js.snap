// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should render correctly 1`] = `
<div
  className="card text-center not-found"
>
  <div
    class="card-header"
  />
  <div
    className="card-body"
  >
    <h2
      className="card-title"
    >
      Page Not Found
    </h2>
  </div>
  <div
    className="card-footer text-muted"
  />
</div>
`;
